yt_dlp/extractor/pbs.py

```diff
@@ -696,3 +697,61 @@ def extract_redirect_urls(info):
             'subtitles': subtitles,
             'chapters': chapters,
         }
+
+
+class PBSKidsIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?pbskids\.org/video/(?P<channel>[\w-]+)/(?P<id>\d+)'
```

Comment:
we are capturing `channel` and then not doing anything with it

we could extract the value

```py
    def _real_extract(self, url):
        video_id, channel = self._match_valid_url(url).group('id', 'channel')
```

and then use it as a `channel` fallback in case traversal doesn't return one

```py
        return {
            'id': video_id,
            'formats': formats,
            'subtitles': subtitles,
            'channel': channel,
            **traverse_obj(meta, {
```

&nbsp;
or you could just remove the capture group from `_VALID_URL`

Reply:
Updated to remove the extra capture.  I think starting simple is the way to go.  There are some features I'd like to explore going forward to pull out full episodes vs clips based on arguments to the extractor, but I'm not there yet. 

Thanks for the help!